### DISCUSSION TITLE
TFT_eSPI LCD does not work

### DISCUSSION THREAD

**Discussion by gcrha** [Bodmer/TFT_eSPI#1178] [Compatible displays and setup files]
Dear Bodmer or other experts,
I start with an some home automatization project where I implement some sensors over my backyard and around the house and now I need net weather station. After I search I found your excellent project with Open Weather and 3.5 inch RPI TFT LCD.
I was buy a one for the test and carefully follow up your instruction as you describe in Arduino forum in 2020 but unsuccessfully.
I tried with many different setup and spent a lot of hours and days tried to resolve my problem before i decided to contact you.
I appreciated your help and any advice about connection and setup.
Picture of the LCD attached as Setup files.

TFT_eSPI ver = 2.3.61
Processor    = ESP8266
Frequency    = 80MHz
Voltage      = 3.32V
Transactions = No
Interface    = SPI
SPI overlap  = No

Display driver = 9486
Display width  = 320
Display height = 480

TFT_CS   = PIN_D8
TFT_DC   = PIN_D3
TFT_RST  = PIN_D4

Font GLCD   loaded
Font 2      loaded
Font 4      loaded
Font 6      loaded
Font 7      loaded
Font 8      loaded
Smooth font enabled

Display SPI frequency = 20.00

Thank you in advance.

Best regards, Goran
[User_Setup.txt](https://github.com/Bodmer/TFT_eSPI/files/6441996/User_Setup.txt)
[User_Setup_Select.txt](https://github.com/Bodmer/TFT_eSPI/files/6441997/User_Setup_Select.txt)
![IMG_3337](https://user-images.githubusercontent.com/75163234/117460196-8c3b3d80-af4c-11eb-9045-7207929c7661.jpg)
![IMG_3336](https://user-images.githubusercontent.com/75163234/117460202-8e9d9780-af4c-11eb-9e55-059d674b9b0a.jpg)
![IMG_3335](https://user-images.githubusercontent.com/75163234/117460206-8fcec480-af4c-11eb-95ec-867fa147aa39.jpg)

**Comment by Bodmer** [comment]
Some users have reported that certain RPi displays must also be provided with 3.3V.

List all the connections you have made. The [recommended pins to use are here](https://github.com/Bodmer/TFT_eSPI/blob/master/Tools/RPi_TFT_Connections.png).

Note that due to design variations not all RPi displays work with the library.

**Reply by gcrha** [reply]
Thanks a lot for the answer.
I also connect 3V3 rail and try any posaible variation of driver. Once when I check driver and connections I get clear picture but once only.
I really dont know what is wrong with this. Display is from Aliexpress just like on the pic and ordered as you prefered in yout description.

**Reply by gcrha** [reply]
The best I get is like on the pic I attached. ![image](https://user-images.githubusercontent.com/75163234/118180180-8ce53f80-b436-11eb-97f5-eef6878fa352.jpeg)
Any suggestion?
